sos/plugins/samba.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
         self.add_cmd_output([
             "wbinfo --domain='.' -g",
             "wbinfo --domain='.' -u",
+            "testparm -s",
             "testparm -s -v"
```

Comment:
@bmr-cymru you are right. I see no value in it. the defaults are determined at compile-time.  I kept it because s/o must have seen value in it... But I am happy to remove it.

Should I add another patch for removal or should I rather amend and push over?